### DISCUSSION TITLE
New Packer Lazy Loading

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1301] [Announcements]
With the new v2.6.5 release of AstroNvim, we are now lazy loading Packer to improve startup time. This does cause some weird migration issues with Packer confusing itself after doing the v2.6.5 update and may lead to users having `:PackerSync` command not found errors.

To resolve these issues, you simply need to clear the cache folder for Neovim (`~/.cache/nvim`) and remove your `packer_compiled.lua` file (Typically in `~/.local/share/nvim/packer_compiled.lua` by default, but may exist in `~/.config/nvim/lua/packer_compiled.lua` or `~/.config/nvim/plugin/packer_compiled.lua`. Please check your `~/.config/nvim` folder for any `packer_compiled.lua` files.)

```lua
rm ~/.local/share/nvim/packer_compiled.lua
rm -rf ~/.cache/nvim
```

If you still run into issues, as a last resort, you can also do a full refresh:

```
rm -rf ~/.local/state/nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.cache/nvim
```

After each of these steps, restart AstroNvim and run :PackerSync.

Sorry again for this inconvenience! This shouldn't happen again since we won't be moving packer again. disappointed

**For more discussion see:** #1294

**Comment by DavidBChristensen** [comment]
Oh man, thanks for writing this up! I thought I was going bonkers.
Removing the packer_compiled.lua and cache fixed the not found error for me.
